google-account-plugin/resources/META-INF/plugin.xml

```diff
@@ -57,6 +57,8 @@
                         serviceImplementation="com.google.cloud.tools.intellij.DefaultAccountPluginConfigurationService"/>
     <applicationService serviceInterface="com.google.cloud.tools.intellij.login.GoogleLoginService"
                         serviceImplementation="com.google.cloud.tools.intellij.login.IntellijGoogleLoginService"/>
+    <applicationService serviceInterface="com.google.cloud.tools.intellij.analytics.IdeaUsageTracker"
```

Comment:
Hmm... I thought Android would go a separate way from here, since it will use its own GA property. We can't use the cloud-reporting lib for Android Studio. There should be different facilities for AS.

What could be the best way?